octobot_trading/channels/trades.py

```diff
@@ -27,6 +27,8 @@ async def push(self, trades, old_trade=False):
     async def perform(self, trades, old_trade=False):
         try:
             for trade in trades:
+                if not trade:
```

Comment:
a trade can be None ?

Reply:
Yes bitmex can response a trade history containing None like : `[TradeA, TradeB, None, TradeC]`. I don't know if it's an issue with ccxt but it happens